hydra-smb.c

```diff
@@ -1311,7 +1311,7 @@ int32_t start_smb(int32_t s, char *ip, int32_t port, unsigned char options, char
   } else if (SMBerr == 0x000071) { /* password expired */
     if (verbose)
       fprintf(stderr, "[%d][smb] Host: %s Account: %s Error: PASSWORD EXPIRED\n", port, ipaddr_str, login);
```

Comment:
I think "Error" in that line is then not helpful? Maybe change it to "Information"?

Reply:
Absolutely, "Information" is much better.